config/config.go

```diff
@@ -14,6 +16,7 @@ type Config struct {
 	StartPort  int               `toml:"start-port"`
 	EndPort    int               `toml:"end-port"`
 	Users      map[string]string `toml:"users"`
+	log        zap.Config        `toml:"log"`
```

Comment:
What is this `log` field for? Maybe we should add a config field like `log_level` and used to parse zap logger level.

Reply:
Let's keep it simple first.

Until we plan to release v1.0.0, we just hardcode log level to `debug` and don't support setting log level. So that we can focus on more important things.